host/src/patches/MurderScene.cpp

```diff
@@ -75,23 +75,49 @@ LONG WINAPI MurderSceneVEH(PEXCEPTION_POINTERS ptrs)
 		bool found = false;
 
 		{
+			printf("Warning: Exception catched! Trying to prevent...\n");
```

Comment:
There's no console attached at all - how is this going to output anything? Also - typo...